qiskit/algorithms/factorizers/shor.py

```diff
@@ -57,10 +57,9 @@ class Shor:
     """
 
     @deprecate_function(
-        """The Shor class is deprecated as of Qiskit Terra 0.22.0 and will be removed
-        no sooner than 3 months after the release date.
-        It is replaced by the tutorial at https://qiskit.org/textbook/ch-algorithms/shor.html
-        """,
+        "The Shor class is deprecated as of Qiskit Terra 0.22.0 and will be removed "
```

Comment:
```suggestion
        "The Shor class is deprecated and will be removed "
```

Reply:
This isn't a good change yet -- this is the old `@deprecate_function` decorator which doesn't insert `since` in the message. It doesn't generate the message at all.

A follow up PR will convert this to the new `@deprecate_func`